Src/PerformanceCollector/Shared/Implementation/WebAppPerformanceCollector/ICounterValue.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation
+{
+    using Microsoft.ApplicationInsights.DataContracts;
+
+    /// <summary>
+    /// Interface represents the counter value.
+    /// </summary>
+    public interface ICounterValue
```

Comment:
This goes on par with other collector. I just made it consistent with other collector. Once if everything is working, we can make a quick change/update to how we collect. Thanks!